fw/config.c

```diff
@@ -0,0 +1,36 @@
+#define CONFIG_VERSION 0
+#define CONFIG_BASE_ADDR (uint8_t*)2
+#define CONFIG_VERSION_ADDR (CONFIG_BASE_ADDR + 0)
+#define CONFIG_REVERSE_TT_ADDR (CONFIG_BASE_ADDR + 1)
+
+#define MAGIC 0xBEEF
```

Comment:
excellent

Reply:
I'm particularly satisfied that the magic bytes can be `0xBEEF` lol